src/main/java/co/inventorsoft/mailsecurity/repositories/EmailRepositoryJdbcImpl.java

```diff
@@ -0,0 +1,127 @@
+package co.inventorsoft.mailsecurity.repositories;
+
+import co.inventorsoft.mailsecurity.models.Email;
+import org.springframework.stereotype.Repository;
+
+import javax.sql.DataSource;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Repository
+public class EmailRepositoryJdbcImpl implements EmailRepository {
+
+    private Connection connection;
+
+    public EmailRepositoryJdbcImpl(DataSource dataSource) {
+        try {
+            this.connection = dataSource.getConnection();
+        } catch (SQLException e) {
+            throw new IllegalStateException(e);
+        }
+    }
+
+    @Override
+    public List<Email> findAll() {
+        String SQL_SELECT_ALL = "SELECT * FROM emails";
+        try {
+            List<Email> emails = new ArrayList<>();
+            Statement statement = connection.createStatement();
```

Comment:
You do not close _Connection_ - this is really bad approach!
Resources should be released when they are not needed.